src/thread/mutex.cpp

```diff
@@ -15,7 +15,7 @@ namespace fc {
   mutex::~mutex() {
     if( m_blist ) 
     {
-      context* c = m_blist;
```

Comment:
I'd recommend that move it to inside `#if 0`, although I don't know why there is a `#if 0`.

Reply:
Must be some long forgotten debug stuff

Reply:
bitshares-fc definitely needs to be cleaned up

Reply:
I'd remove the whole #if 0 block. Doesn't really readability.